src/App.jsx

```diff
@@ -12,24 +12,78 @@ import AdminDashboard from './pages/adminDashboard';
 import VendorDashboard from './pages/vendorDashboard';
 import CartPage from './pages/CartPage';
 import PageNotFound from './components/PageNotFound';
-import { UpdateProduct } from './pages/UpdateProduct';
-import CreateProduct from './pages/Createproduct';
+import Topnav from './components/Landingpage/topnav';
+import Footer from './components/Landingpage/Footer';
 
 function App() {
 	return (
 		<BrowserRouter>
 			<Routes>
 				<Route exact path="/" element={<HomePage />} />
-				<Route path="/buyer" element={<BuyerSignupPage />} />
-				<Route path="/vendor" element={<VendorSignupPage />} />
+				<Route
+					path="/buyer"
+					element={
+						<>
+							<Topnav displaySearchBar />
+							<BuyerSignupPage />
+							<Footer />
+						</>
+					}
+				/>
+				<Route
+					path="/vendor"
+					element={
+						<>
```

Comment:
what if you put those components on the landing page, and import single file

`
<Route
					path="/buyer"
					element={
						<>
							<Topnav displaySearchBar />
							<BuyerSignupPage />
							<Footer />
						</>
					}
				/>
`